packages/xl-docx-exporter/src/docx/docxExporter.ts

```diff
@@ -111,8 +111,25 @@ export class DOCXExporter<
     nestingLevel = 0
   ): Promise<Array<Paragraph | Table>> {
     const ret: Array<Paragraph | Table> = [];
+    let currentNumberingStart = 1;
+    let numberingInstanceCount = 0;
+    let isFirstNumberedListItem = false;
 
     for (const b of blocks) {
+      if (b.type === "numberedListItem") {
```

Comment:
This code is a bit messy and could definitely be improved, but its the best I could come up with to ensure a unique instance id is given to each numbered list. In theory this id should guarantee no weird continuations of non adjacent lists happen, but for some reason they still are. I would not merge this as is